var/spack/repos/builtin/packages/cardioid/package.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import spack.environment as ev
+import os
+
+
+class Cardioid(CMakePackage):
+    """Cardiac simulation suite."""
+
+    homepage = 'https://baasic.llnl.gov/comp-bio/cardioid-code.php'
+    git = "https://github.com/LLNL/cardioid.git"
+    maintainers = ['rblake-llnl']
+
+    version('master', branch='master')
```

Comment:
the deveopment version of a package is special-named `develop` in spack: `version('develop', branch='master')` would be the spelling